components/ActivityCardDetails.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { activities } from "@/lib/dummydata";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+
+const StyledHeadline = styled.h2`
+    text-align: center;
+`;
+
+const StyledButton = styled.button`
+    position: absolute;
+    left: 8%;
+    padding: 1% 3%;
+    border: 1px solid black;
+    border-radius: 10px;
+    background-color: transparent;
+`
+
+const StyledLink = styled(Link)`
+    text-decoration: none;
+    color: black;
+`
+
+const StyledSection = styled.section`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 8vw;
+    margin-right: 8vw;
+    margin-top: 15vw;
```

Comment:
working with vw can be very brittle and causes overflows regularly. You should consider using rem or px for margins instead.

Reply:
Done ✅